packages/core/src/routes/session.test.ts

```diff
@@ -221,6 +285,99 @@ describe('sessionRoutes', () => {
     });
   });
 
+  describe('POST /session/sign-in/social', () => {
+    it('sign-in with social and redirect', async () => {
+      const response = await sessionRequest.post('/session/sign-in/social').send({
+        connectorId: 'social_enabled',
+        state: 'state',
+        redirectUri: 'logto.dev',
+      });
+      expect(response.body).toHaveProperty('redirectTo');
```

Comment:
nit: could give the expected Property value.

toHaveProperty(key, value)

Reply:
fixed.